core/sail/lucene/src/test/java/org/eclipse/rdf4j/sail/lucene/examples/LuceneSailExample.java

```diff
@@ -69,45 +75,35 @@ public static void createSimple() throws Exception {
-					+ "BINDINGS ?type { \n" + " (<http://www.w3.org/2002/07/owl#Class>) \n" + "}";
-			tupleQuery(queryString, connection);
-
-			// search for property "name" with domain "person"
-			queryString = "PREFIX search: <" + LuceneSailSchema.NAMESPACE + "> \n"
-					+ "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> \n" + "SELECT * WHERE { \n"
-					+ "?subject rdfs:domain ?domain . \n" + "?subject search:matches ?match . \n"
-					+ "?match search:query \"chat\" ; \n" + "       search:score ?score . \n"
-					+ "?domain search:matches ?match2 . \n" + "?match2 search:query \"person\" ; \n"
-					+ "        search:score ?score2 . \n" + "} LIMIT 5";
+			// String queryString = "PREFIX geof: <http://www.opengis.net/def/function/geosparql/> PREFIX geo:
+			// <http://www.opengis.net/ont/geosparql#> PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> PREFIX
+			// rdfs: <http://www.w3.org/2000/01/rdf-schema#> SELECT ?id WHERE { ?s <http://nuts.de/geometry> ?o . FILTER
+			// (geof:sfWithin(\"Point(-2.7633 47.826)\"^^geo:wktLiteral,?o)) }";
+			// String queryString = "PREFIX geof: <http://www.opengis.net/def/function/geosparql/> PREFIX geo:
+			// <http://www.opengis.net/ont/geosparql#> PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> PREFIX
+			// rdfs: <http://www.w3.org/2000/01/rdf-schema#> SELECT ?id WHERE { ?s <http://nuts.de/geometry> ?o . FILTER
+			// (geof:sfContains(?o,\"POINT(33.30260 38.675310)\"^^geo:wktLiteral)) ?s <http://example.com/id> ?id . }";
+			// String queryString = "PREFIX geof: <http://www.opengis.net/def/function/geosparql/> PREFIX geo:
+			// <http://www.opengis.net/ont/geosparql#> PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> PREFIX
+			// rdfs: <http://www.w3.org/2000/01/rdf-schema#> SELECT ?id WHERE { ?s <http://nuts.de/geometry> ?o . FILTER
+			// (geof:sfWithin(\"Point(7.98 45.363)\"^^geo:wktLiteral,?o)) ?s <http://nuts.de/id> ?id . }";
+			String queryString = "SELECT ?s0 where { ?s0 <https://linkedopendata.eu/prop/direct/P127> ?coordinates . FILTER ( <http://www.opengis.net/def/function/geosparql/distance>(\"Point(12.2018 44.4161)\"^^<http://www.opengis.net/ont/geosparql#wktLiteral>,?coordinates,<http://www.opengis.net/def/uom/OGC/1.0/metre>)< 100000) .    ?s0 <https://linkedopendata.eu/prop/direct/P35> <https://linkedopendata.eu/entity/Q9934> .}";
+			/*
+			 * String queryString = "PREFIX geo: <http://www.opengis.net/ont/geosparql#>\n" +
+			 * "PREFIX geof: <http://www.opengis.net/def/function/geosparql/>\n" +
+			 * "PREFIX uom: <http://www.opengis.net/def/uom/OGC/1.0/>\n" + "PREFIX ex: <http://example.org/>\n" +
+			 * "SELECT *\n" + "WHERE {\n" + "  ?lmA a ex:Landmark ;\n" +
+			 * "       geo:hasGeometry [ geo:asWKT ?coord1 ].\n" + "  ?lmB a ex:Landmark ;\n" +
+			 * "       geo:hasGeometry [ geo:asWKT ?coord2 ].\n" +
+			 * "  BIND((geof:distance(?coord1, ?coord2, uom:metre)/1000) as ?dist) .\n" +
+			 * "  FILTER (str(?lmA) < str(?lmB))\n" + "}";
+			 */
```

Comment:
This looks like you've used the example for your own tests - which is fine of course, but I think it needs some cleaning up.